extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -270,7 +270,8 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(this.model.server.providerName, azdata.DataProviderType.QueryProvider);
 
-		let query = `SELECT name FROM sys.schemas`;
+		const escapedQuotedDb = this.databaseDropdown.value ? (<azdata.CategoryValue>this.databaseDropdown.value).name.replace(/]/g, ']]') : '';
+		const query = 'SELECT name FROM ${escapedQuotedDb}.sys.schemas';
```

Comment:
The period should actually also be in the previous part (we don't want to include the period if there isn't a DB name selected)

Reply:
Sorry about that. I overlooked that. Updated.